functions.yml

```diff
@@ -407,16 +428,17 @@ gwas_functions:
           [Row(beta=1.1658962684583645, oddsRatio=3.208797540870915, waldConfidenceInterval=[0.2970960052553798, 34.65674891673014], pValue=0.2943946848756771)]
       args:
         - name: genotypes
-          doc: An array of genotypes
+          doc: An numeric array of genotypes
         - name: phenotypes
-          doc: An array of phenotype values
+          doc: A double array of phenotype values
         - name: covariates
-          doc: a matrix of covariates
+          doc: A ``spark.ml`` ``Matrix`` of covariates
         - name: test
           doc: Which logistic regression test to use. Can be 'LRT' or 'Firth'
```

Comment:
Can we wrap `'LRT'` and `'Firth'` in backticks?

Reply:
Done